docs/source/anesthetic.rst

```diff
@@ -83,6 +84,15 @@ anesthetic.scripts module
    :show-inheritance:
 
 
+anesthetic.tension module
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. automodule:: anesthetic.tension
+   :members:
+   :undoc-members:
+   :show-inheritance:
```

Comment:
I've added the tension module to the docs. Can be found here: https://anesthetic.readthedocs.io/en/tension/anesthetic.html#module-anesthetic.tension